node/core/approval-voting/src/persisted_entries.rs

```diff
@@ -76,13 +76,13 @@ impl From<TrancheEntry> for crate::approval_db::v1::TrancheEntry {
 /// particular block.
 #[derive(Debug, Clone, PartialEq)]
 pub struct ApprovalEntry {
-	tranches: Vec<TrancheEntry>,
-	backing_group: GroupIndex,
-	our_assignment: Option<OurAssignment>,
-	our_approval_sig: Option<ValidatorSignature>,
+	pub(super) tranches: Vec<TrancheEntry>,
```

Comment:
Why did this stuff all become `pub(super)`?

Reply:
There were `impl From` implementations which would be sufficient to convert between `approval_db::v1` types and these types.